Gemfile.lock

```diff
@@ -263,6 +263,3 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   vcr
   webmock
-
-BUNDLED WITH
```

Comment:
You will want to run

Gem update bundler 

To upgrade your bundler to the latest version. In the mean time, can you add this content back in